product-service/api.yaml

```diff
@@ -21,8 +22,27 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/Product'
+    put:
```

Comment:
generally put is used only for update operation and it some kind of best practice to use post for create operation

Reply:
Totally agree, just wanted it to be integrated with FE App where PUT is used mistakenly for both creation and modification operations